ReleaseProcessAutomation.IntegrationTests/Jira/JiraCombinedIntegrationTests.cs

```diff
@@ -421,7 +421,7 @@ public void FindAllVersionsWithVersionNumber_FindsProperVersions ()
   }
 
   [Test]
-  public void FindIssuesWithOnlyExactFixVersion_ReturnsFoundVersions ()
+  public void FindIssuesWithOnlyExactFixVersion_WithSeveralWrongIssues_ReturnsSingleCorrectIssue ()
```

Comment:
`ReturnsSingleCorrectIssue ` => the number of correct issues isn't really important, is it? i.e. we don't know or expect there to be a single correct issue and it's also not derived from the other parts of the test name. Do you mean `ReturnsOnlyCorrectIssues`? 

also `CorrectIssue` and `WrongIssue` =>please qualify, i.e. is it issues that have the correct/incorrect version? And what does this mean?

You should be able to deduce the basic behavior of the code just by reading the names (and knowing a bit about the indended behavior of the tool). for me, the current name is a bit too far out there, so I can only inject my own interpretation into the test name, which isn't a good thing.